metadata-service/auth-impl/src/main/java/com/datahub/authentication/token/StatefulTokenService.java

```diff
@@ -78,17 +79,20 @@ public String generateAccessToken(@Nonnull final TokenType type, @Nonnull final
       @Nonnull final String name, final String description, final String actorUrn) {
     Date date = new Date();
     long timeMilli = date.getTime();
-    return generateAccessToken(type, actor, DEFAULT_EXPIRES_IN_MS, timeMilli, name, description, actorUrn);
+    return generateAccessToken(type, actor, Optional.of(DEFAULT_EXPIRES_IN_MS), timeMilli, name, description, actorUrn);
   }
 
   @Nonnull
   public String generateAccessToken(@Nonnull final TokenType type, @Nonnull final Actor actor,
-      @Nonnull final long expiresInMs, @Nonnull final long createdAtInMs, @Nonnull final String tokenName,
+      @Nonnull final Optional<Long> expiresInMs, @Nonnull final long createdAtInMs, @Nonnull final String tokenName,
```

Comment:
Why not just make this a @Nullable Long? 

Usually we don't use Optionals over nulls in function parameters